src/components/Tabs/TabsItem/TabsItem.tsx

```diff
@@ -43,7 +43,15 @@ export const TabsItem: React.FC<TabsItemProps> = ({
     }, [hint, title]);
 
     return (
-        <div className={b({active, disabled, direction})} title={htmlTitle} onClick={handleClick}>
+        <div
+            role="tab"
+            aria-selected={active === true}
+            aria-disabled={disabled === true}
```

Comment:
These properties are exactly what I told you about recently when we were talking about improvements of a11y in common components